tethysapp/metdataexplorer/templates/metdataexplorer/modals/modalInformationAboutGroup.html

```diff
@@ -2,7 +2,12 @@
     <div class="modal-dialog">
         <div class="modal-content myModal">
             <div class="modal-header">
-                <button type="button" class="close" data-dismiss="modal">&times;</button>
+                <h4>Information about Catalog</h4>
+                <!--<div id="pop-up_description2">
+                </div>-->
```

Comment:
What is going on here?

Reply:
Oh that was working out how to get the close button at the correct place and left a trial item in a comment.